flic2Button.js

```diff
@@ -1,8 +1,8 @@
 // event emitter
 import EventEmitter from 'react-native/Libraries/vendor/emitter/EventEmitter';
 
-// local
-import Flic2 from './index.js';
+// the Flic2 module
+import Flic2 from 'react-native-flic2';
```

Comment:
internal package mag naar zichzelf verwijzen

```suggestion
import Flic2 from './index.js';
```